api/v1beta1/machinehealthcheck_webhook.go

```diff
@@ -165,8 +176,12 @@ func (m *MachineHealthCheck) validate(old *MachineHealthCheck) error {
 		)
 	}
 
-	if len(allErrs) == 0 {
-		return nil
+	if len(m.Spec.UnhealthyConditions) == 0 {
+		allErrs = append(allErrs, field.Forbidden(
+			specPath.Child("unhealthyConditions"),
+			"must have at least one value",
```

Comment:
```suggestion
			"must have at least one entry",
```
up to you, not sure what sounds/fits better